tensorflow_datasets/text/__init__.py

```diff
@@ -26,3 +26,5 @@
 from tensorflow_datasets.text.squad import Squad
 from tensorflow_datasets.text.wikipedia import Wikipedia
 from tensorflow_datasets.text.xnli import Xnli
+from tensorflow_datasets.text.yelp_polarity import YelpPolarityReviews
+from tensorflow_datasets.text.yelp_polarity import YelpPolarityReviewsConfig
```

Comment:
You must delete the `YelpPolarityReviewsConfig` file here.

Reply:
Sorry for the wrong information. Your way was true, you should import `YelpPolarityReviewsConfig`.